tests/resources/VCH-Util.robot

```diff
@@ -717,6 +717,17 @@ Install VIC with version to Test Server
     Set Environment Variable  INITIAL-VERSION  ${version}
     Run  rm -rf vic.tar.gz vic
 
+Install VIC release to Test Server
```

Comment:
we don't need a whole new keyword here, the existing one is bad to begin with.  I would rather get this PR merged - #6893 (which I have been trying to merge for months), then rebase your PR into using the fixed existing  Install VIC with version to Test Server keyword on the new test.  otherwise, this LGTM

Reply:
I agree with the approach in #6893; it's definitely the right long-term approach. If that goes in soon, I'm happy to rebase to pick up that change. However, I don't think it's necessary to block this feature merge on that PR.

Reply:
Updated!